src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -372,6 +368,74 @@ object DatabaseConversions {
+        // so values below are going to be discarded during transformation Operation -> OperationRow
+        from match {
+          case reveal: InternalOperationResults.InternalRevealResult =>
+            reveal
+              .into[Reveal]
+              .withFieldConst(_.counter, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.fee, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.gas_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.storage_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.metadata, ResultMetadata[OperationResult.Reveal](reveal.result, List.empty, None))
+              .transform
+          case transaction: InternalOperationResults.InternalTransactionResult =>
+            transaction
+              .into[Transaction]
+              .withFieldConst(_.counter, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.fee, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.gas_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.storage_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(
+                _.metadata,
+                ResultMetadata[OperationResult.Transaction](
+                  transaction.result,
+                  transaction.result.balance_updates.getOrElse(List.empty),
+                  None
+                )
+              )
+              .transform
+
+          case origination: InternalOperationResults.InternalOriginationResult =>
+            origination
+              .into[Origination]
+              .withFieldConst(_.counter, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.fee, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.gas_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.storage_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(
+                _.metadata,
+                ResultMetadata[OperationResult.Origination](
+                  origination.result,
+                  origination.result.balance_updates.getOrElse(List.empty),
+                  None
+                )
+              )
+              .transform
+
+          case delegation: InternalOperationResults.InternalDelegationResult =>
+            delegation
+              .into[Delegation]
+              .withFieldConst(_.counter, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.fee, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.gas_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(_.storage_limit, InvalidPositiveDecimal("Discarded"))
+              .withFieldConst(
+                _.metadata,
+                ResultMetadata[OperationResult.Delegation](delegation.result, List.empty, None)
+              )
+              .transform
+        }
+    }
+
+  def appendToFile(str: String) =
```

Comment:
Leftover?

Reply:
yes! I thought I got rid of all debugging 😄